### PR TITLE
ffda-domain-director: make /etc/config/ffda available on initial setup

### PR DESCRIPTION
**Pull request by belzebub40k**
Currently it is not possible to disable the domain-director package during initial setup as _/etc/config/ffda_ doesn't exist at this point. A user might think he/she disabled it but didn't. An other problem with the current version is that the "Enabled" checkbox is unchecked by default and so a user could think this feature is opt-in and not opt-out.

Tested initial setup and sysupgrade.

### CODE REVIEW

**Comment by blocktrron** [comment]
@belzebub40k Is your Gluon image compiled with Multidomain support enabled? I've just tested this on a fresh x86 image and the config file is created correctly on first setup-mode boot.

**Comment by belzebub40k** [comment]
Yes, it is compiled with Multidomain support enabled. Let me try again. I tested it with a TL-WR842v3.

**Comment by belzebub40k** [comment]
Ok, it is a bit different than I wrote. The problem is not the missing file it is the missing `director` section. Because it is not present at this point `ffda.director.enabled` couldn't be set.

**Comment by belzebub40k** [comment]
Ok, I think I understood why it happens. I've set the domain-director to disabled in the site.conf and enabled it explicitly in a subset of domain. But the upgrade script terminates before committing the uci changes to ffda when it is disabled. I guess I should try it the other way round. Enabled by default and disable it in the domains where we don't want it at the moment.

EDIT: Tried this and it worked as expected.

**Comment by belzebub40k** [comment]
So simply moving [this uci:commit](https://github.com/freifunk-darmstadt/ffda-packages/blob/master/ffda-domain-director/luasrc/lib/gluon/upgrade/950-ffda-domain-director#L58) in front of the [ddutil.is_enabled_site](https://github.com/freifunk-darmstadt/ffda-packages/blob/master/ffda-domain-director/luasrc/lib/gluon/upgrade/950-ffda-domain-director#L35) check should also help to fix this corner case.
